tests/device_tests/test_reset_backup_bip39.py

```diff
@@ -0,0 +1,167 @@
+# This file is part of the Trezor project.
```

Comment:
Let's call this file `test_reset_delayed_backup_bip39` and the other analogically?

Reply:
I also think we can parametrize these tests.